app/models/campaign.rb

```diff
@@ -48,6 +48,12 @@ def self.end_at(date)
       .distinct
   end
 
+  # @param [Array<String>] tags
+  # @return [Campaign]
+  def self.tag_matches(tags)
```

Comment:
@harold93 what happens is tags if :blank?

Reply:
it returns an empty array